test/Player-test.js

```diff
@@ -1,3 +1,11 @@
 const chai = require('chai');
 const expect = chai.expect;
-const Player = require('../js/Player.js');
\ No newline at end of file
+const Player = require('../js/Player.js');
+
+
+describe('Player', function () {
```

Comment:
I'm glad we could go over this example of testing together. Thanks for showing me.